BasicSample/Packages/manifest.json

```diff
@@ -4,8 +4,9 @@
     "com.microsoft.mixedreality.openxr.sampleshared": "file:../../SampleShared",
     "com.microsoft.mixedreality.toolkit.foundation": "file:../../MixedRealityPackages/com.microsoft.mixedreality.toolkit.foundation-2.8.3.tgz",
     "com.microsoft.mixedreality.toolkit.standardassets": "file:../../MixedRealityPackages/com.microsoft.mixedreality.toolkit.standardassets-2.8.3.tgz",
-    "com.unity.ide.rider": "3.0.16",
-    "com.unity.ide.visualstudio": "2.0.16",
+    "com.microsoft.mrtk.graphicstools.unity": "https://github.com/microsoft/MixedReality-GraphicsTools-Unity.git?path=/com.microsoft.mrtk.graphicstools.unity",
```

Comment:
Good point.  For a single point of time, the hash of commit of given repo is deterministic.  But over time without a tag it can be hard to reproduce the feature or quality.  I will look into this.